### ISSUE TITLE
Google Maps: Javascript Error after update to Page Builder2.5.7 (and other versions)

### ISSUE THREAD

**Issue by vollyimnetz** [siteorigin/so-widgets-bundle#497]
With the current update to 2.5.7 i got the following error. 

```
TypeError: Cannot read property '0' of null at Object.sowb.onLoadMapsApiError [as error]
...
at _.Le.setDraggable
```
I need to change something in the maps-plugin and renew save the page to get the plugin to work again.

I disabled "scroll to zoom" on all of my plugins on serveral pages - so they do not use this option which is enabled by default.

**This is the second time i have this bug after an update to a newer page-builder version.**

**Comment by braamgenis** [comment]
Hi @vollyimnetz Thanks for reporting. We're aware of this issue and should have it resolved in the next update due out very soon.

**Comment by braamgenis** [comment]
Fixed in 5231b6ed40da1353b9ebab2dda2188105b43372a
